### PR TITLE
v2v: move v2v-vmware ConfigMap to kubevirt-hyperconverged namespace

### PR DESCRIPTION
**Pull request by mareklibra**
More safe to avoid unrelated modifications.

Follow-up for #507

### CODE REVIEW

**Comment by mareklibra** [comment]
Cc @jwmatthews

File: src/k8s/requests/v2v/startV2VvmwareController.js [hunk 1]
```diff
@@ -35,22 +35,36 @@ export const startV2VVMWareController = async ({ namespace }, { k8sGet, k8sCreat
     throw new Error('V2V VMWare: namespace must be selected');
   }
 
+  const name = V2VVMWARE_DEPLOYMENT_NAME;
```
Comment (rawagner) [inline src/k8s/requests/v2v/startV2VvmwareController.js]: isnt this unnecessary ? why not use `V2VVMWARE_DEPLOYMENT_NAME` directly, instead of `name` ?

Comment (mareklibra) [reply src/k8s/requests/v2v/startV2VvmwareController.js]: good point, I can change

Comment (suomiy) [reply src/k8s/requests/v2v/startV2VvmwareController.js]: The name is used in multiple places - I would rather leave it like it is

**Comment by jwmatthews** [comment]
cc @jmontleon

**Pull request merged**